packages/src/components/editable/editable.component.ts

```diff
@@ -225,17 +268,19 @@ export class SlateEditableComponent implements OnInit, OnChanges, OnDestroy, Aft
         const window = AngularEditor.getWindow(this.editor);
         this.addEventListener(
             'selectionchange',
-            event => {
-                this.toSlateSelection();
```

Comment:
toSlateSelection is my redefined name to update the slate selection based on the browser selection, I think its semantics are very clear, and toNativeSelection is a reverse process, no need to change the name.